esphome/components/esp32/preferences.cpp

```diff
@@ -36,6 +36,7 @@ class ESP32PreferenceBackend : public ESPPreferenceBackend {
     save.key = key;
     save.data.assign(data, data + len);
     s_pending_save.emplace_back(save);
+    ESP_LOGV(TAG, "s_pending_save: key: %s, len: %d", key.c_str(), len);
```

Comment:
```suggestion
    ESP_LOGVV(TAG, "s_pending_save: key: %s, len: %d", key.c_str(), len);
```
Can we move these to very verbose setting? This would appear to spam the logs somewhat and most developers will have verbose level turned on.